modules/local/starfusion/build/environment.yml

```diff
@@ -4,7 +4,8 @@ channels:
 dependencies:
   - bioconda::dfam=3.7
   - bioconda::hmmer=3.4
-  - bioconda::star-fusion=1.7.0
-  - bioconda::trinity=2.15.2
-  - bioconda::samtools=1.21
-  - bioconda::star=2.7.11b
+  - bioconda::minimap2=2.28
```

Comment:
I will re-test this just in case and adjust the environment if necessary.

Reply:
I just did and minimap2 is necessary

Reply:
Yes, otherwise it fails without minimap2:

![image](https://github.com/user-attachments/assets/67924de1-e583-41b1-a630-a6e27d817a47)

```
    - *** ERROR, cannot locate required tool in PATH setting: minimap2 ***
```